crates/swc_ecma_transforms_testing/src/lib.rs

```diff
@@ -180,6 +184,7 @@ impl Tester<'_> {
                 },
             )?;
 
+        let tr = tr(self);
```

Comment:
Delaying `tr`'s execution until after the parser runs allows comments to be read correctly.
I fixed this in one place, but it seems other cases were missed where comments still come up empty.